Doc/license.rst

```diff
@@ -703,7 +703,7 @@ of the OpenSSL license here::
      * The implementation was written so as to conform with Netscapes SSL.
      *
      * This library is free for commercial and non-commercial use as long as
-     * the following conditions are aheared to.  The following conditions
+     * the following conditions are adhered to.  The following conditions
```

Comment:
You need a special authorization from PSF lawyers for changing the text of the license.

I suggest to leave this text as is.

Reply:
Reverted it back. Thanks.